samples/external-secrets/README.md

```diff
@@ -0,0 +1,100 @@
+# External Secrets
+
+[External Secrets][eso] is an operator to fetch content from secure stores and synthesize that state into a Kubernetes `Secret`.
+
+This sample uses a fake `SecretStore` (to avoid external infrastructure) and `ExternalSecret` to create a binding Secret. The [Spring PetClinic][petclinic] workload and MySQL database are independently bound to the `ExternalSecret`.
+
+> NOTE: Provisioned Service support within External Secrets is a proposal that is not part of any External Secrets release. This sample is to demonstrate what landing provisioned service support within ESO can enable.
+
+## Setup
+
+If not already installed, [install the ServiceBinding CRD and controller][install], and an experimental version of the [External Secrets Operator from PR #2263][pr-2263].
```

Comment:
ESO landed support with the merging of [PR 2263](https://github.com/external-secrets/external-secrets/pull/2263); should we call out that this works with [v0.8.2](https://github.com/external-secrets/external-secrets/releases/tag/v0.8.2) and later?

Reply:
I had run `git push` but didn't notice that it was stuck on an auth prompt.